local/local.go

```diff
@@ -316,8 +316,8 @@ func (i *Image) Save(additionalNames ...string) error {
 	i.inspect = inspect
 
 	var errs []imgutil.SaveDiagnostic
-	for _, n := range additionalNames {
-		if err := i.docker.ImageTag(context.Background(), i.repoName, n); err != nil {
+	for _, n := range append([]string{i.Name()}, additionalNames...) {
```

Comment:
This is one of the main changes we had to make when the ggcr update updated docker. For some reason, adding the original image name to `RepoTags` in the manifest no longer results in the initial image (saved during `doSave()`) getting tagged, so we include the original name here when processing additional tags. We can dig in deeper later to understand better what docker requires during save, but this approach works equally as well as before.